build/builder/mkrelease.sh

```diff
@@ -39,7 +39,7 @@ case "${1-}" in
       XGOARCH=amd64
       XCMAKE_SYSTEM_NAME=Linux
       TARGET_TRIPLE=x86_64-unknown-linux-gnu
-      LDFLAGS="-static-libgcc -static-libstdc++"
+      LDFLAGS="-static-libgcc -static-libstdc++ -lrt"
```

Comment:
Does your patch work for the other targets supported below? What does the `build/builder.sh mkrelease` command says for the various targets?

Reply:
It does work for `darwin`, `windows` and `linux-musl` too. 
`linux-gnueabi` and `linux-msan` don't work for me for the unmodified master branch.